docs/discover/kuery.asciidoc

```diff
@@ -73,3 +73,87 @@ set these terms will be matched against all fields. For example, a query for `re
 in the response field, but a query for just `200` will search for 200 across all fields in your index.
 ============
 
+===== Nested Field Support
+
+KQL supports querying on {ref}/nested.html[nested fields] through a special syntax. Nested fields can be queried in subtly different
+ways depending on the results you would like to see, so crafting nested queries requires some extra thought.
+ 
+One of the main things to consider is how you want parts of the nested query to match the individual nested documents.
+There are two main approaches to take:
+
+1. Parts of the query may only match a single nested document. This is what most people want when querying on a nested field.
+2. Parts of the query may match different nested documents. This is how a regular object field works but nested fields 
```

Comment:
```suggestion
*  *Parts of the query match different nested documents.* This is how a regular object field works. 
```

Reply:
Removing the "may" makes it sound to me like parts of the query *must* match different documents. Do you think it's fair to keep it in?

Reply:
How about "can"?  Otherwise, keep may" in.

Reply:
"can" sounds good, I'll do that 👍